### PR TITLE
push skeleton

### PR DESCRIPTION
**Pull request by cdrainkainos**
Initial push of API skeleton

### CODE REVIEW

**Review by TomaszLaski — CHANGES_REQUESTED** [review]
I think we should add .DS_Store files to .gitignore

**Review by TomaszLaski — APPROVED** [review]

File: src/main/java/com/kainos/ea/database/DatabaseConnection.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package com.kainos.ea.database;
+
```
Comment (boydstepnew) [inline src/main/java/com/kainos/ea/database/DatabaseConnection.java]: Lets remove this as it doesn't belong in a skeleton

File: src/main/resources/banner.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+================================================================================
+
```
Comment (boydstepnew) [inline src/main/resources/banner.txt]: What is this? Doesn't look used, can delete it. If you want to keep the directory - add a blank file

**Review by boydstepnew — CHANGES_REQUESTED** [review]
Need to remove all the .DS files

**Review by BlazejRaj — APPROVED** [review]
looks fine

**Review by boydstepnew — APPROVED** [review]

**Pull request merged**